frontend/playwright.config.ts

```diff
@@ -16,12 +17,12 @@ export default defineConfig({
     { name: "chromium", use: { ...devices["Desktop Chrome"] } },
     { name: "webkit", use: { ...devices["Desktop Safari"] } },
     { name: "firefox", use: { ...devices["Desktop Firefox"] } },
-    // { name: "Mobile Chrome", use: { ...devices["Pixel 5"] } },
-    // { name: "Mobile Safari", use: { ...devices["iPhone 12"] } },
   ],
 
   webServer: {
-    command: `bun run dev --port ${port}`,
+    command: process.env.CI
+      ? `bun run build && bun run start --port ${port}` // production build for CI/Lighthouse
```

Comment:
There is no script in `package.json` named `start`. I think you were referring to `dev`, but that is also not correct because that runs a dev-mode version of the app. What you want here is `bun run build` (builds a production version of the app) then `bun run preview` (serves the built/bundled app).